es/es-repository-azure/build.gradle

```diff
@@ -0,0 +1,7 @@
+dependencies {
+    compile project(':es:es-server')
```

Comment:
Can we use `implementation` and `testImplementation` instead of `compile`/`testCompile`? `compile` is deprecated